### ISSUE TITLE
Loss of genome/seqinfo with tagClustersGR

### ISSUE THREAD

**Issue by snikumbh** [charles-plessy/CAGEr#62]
I noticed that the GRanges returned by `tagClustersGR` does not preserve genome/seqinfo. In comparison, the `CTSScoordinatesGR` preserves it.

This is perhaps related to or a subset of https://github.com/charles-plessy/CAGEr/issues/23

**Comment by charles-plessy** [comment]
Hi Sarvesh,

good catch! At the moment there is no proper `seqinfo` set for the tag clusters stored in the `GRangesList` object at `cageExpObject@metadata$tagClusters`.

A short-term fix could be to add the following lines towards the end of `clusterCTSS` method:

```{r}
seqlevels(ctss.cluster.list) <- seqlevels(CTSStagCountSE(object))
seqinfo(ctss.cluster.list)   <- seqinfo(CTSStagCountSE(object))
```

A more satisfying fix would be to have `.distclu` `.paraclu` and `.predefined.clusters` do the right thing from the start, but that would be a bigger patch.

If we make a last update in this release cycle, I will be happy to take care of this issue.  If not, let's fix it at the beginning of next cycle.  We can easily port the fix to the stable branch.

Cheers,

-- 
Charles

**Comment by snikumbh** [comment]
I think the intermediate functions, `.distclu` etc., do not need this info,
or do they?

If not, I believe there are no downsides to apply and keep the short-term
fix.

Sarvesh

**Comment by snikumbh** [comment]
You are right. Ideally, if they receive it that way they should return it with genome/seqinfo intact. I didn't have access to the codebase when I commented earlier.

We have some days before the next release date. I think we should go for it.

**Comment by charles-plessy** [comment]
Closed in 9f985be270118b7e118237b560f2a37ea6d56f5a

I still do not manage to get the closing syntax correct in commit messages :(
